### ISSUE TITLE
New Logo for Rainloop?

### ISSUE THREAD

**Issue by elioqoshi** [RainLoop/rainloop-webmail#1408]
Hi there,

I was wondering whether you need to have a refresh on your branding for Rainloop? Would love to help. 
Am focused on working with open source projects: https://ura.design 

Would be great to see this happening!

**Comment by extremeshok** [comment]
@elioqoshi  your business strategy is to lurk around and spam your advertisements ?

**Comment by ervee** [comment]
Yes, please edit (remove) your message and close this case. This is ridiculous.

**Comment by extremeshok** [comment]
@ervee Report him: https://github.com/contact/report-abuse?report=elioqoshi

**Comment by elioqoshi** [comment]
Hah, are you serious? I wanted to do this for free. If you would have done your research you would have found out that we are focused on doing branding for open source projects. Have you even checked my background before calling me a spammer? 

Reconsider your attitude before driving people into reporting. Thank you.

**Comment by elioqoshi** [comment]
Even if I was a spammer, having that attitude drives contributors away. And then we complain why there are lack of contributors and inclusivity. Great job driving potential contributors away.

I was recently considering to switch to RainLoop as I really liked the UX. If the attitude of the project is however even remotely close to what I have seen here, I really hope someone forks it into a project where one isn't an asshole at the first best opportunity.

**Comment by giannisk** [comment]
You do realise we have done branding, free of charge, for numerous free and open-source projects, right? If not, have a look at that "spam URL" that Elio posted.

Congratulations for driving us away at first glance. Take care and goodbye.

**Comment by brantje** [comment]
What is this? Do you guy's react to every contributor like this? The guy offers you to help you out with a logo. I don't see why he's spamming because he is offering it for free. I've worked with him great guy in any way, not a spammer!

Also a request to remove / edit the message, so you like censorship?

**Comment by animalillo** [comment]
Woah amazing! I was thinking in setting this up, but this gave me a bad feeling.

**Comment by extremeshok** [comment]
Your work experience consists of "Portfolio Gigs", most of the logo's are mis-representations made to look like your did the logos for those various projects and companies. Facts are they do not correlate to what is written on your CV.
If you really wanted to make logo's and graphics for this project, you would of already made them, and then submitted a PR and/or posted them for comment/voting.

Your modus operandi consists of spamming projects mailing lists and repos in order to solicit for work.

Zero tolerance to spammers.

**Comment by brantje** [comment]
@extremeshok I'm sure he does logo's. He made some logo's for my project and they are better looking than your current one imho.

**Comment by ervee** [comment]
@elioqoshi @giannisk @brantje @animalillo , if you did your research you should know that nor me nor @extremeshok are in any way the autors of this software.

If you want to selflessly contribute, do a PR or contact the author by e-mail. His e-mail is published on the front page for god-sake.

Stop this idiotic spam please.

**Comment by douglascamata** [comment]
I never, in my entire life, saw a worst reaction coming from open-source people. If you guys don't need a logo, just say `no, thanks` and close the case.

Because of bad people, like you, in the open source community, that doesn't know how to deal with other people, which are the core components of the model, that some projects are completely abandoned.

Be nice, BY DEFAULT.

**Comment by brantje** [comment]
Curious why you see my opinions as spam. 
Spam: send the same message indiscriminately to (a large number of Internet users).

**Comment by extremeshok** [comment]
@branje only spammer is the person that created the issue.

@douglascamata people like us are needed... we directly fund open source projects, contribute code and releases. We own commercial rainloop licenses, which keeps this project alive.

**Comment by douglascamata** [comment]
@extremeshok no, people like you are definitely not needed. If it weren't for a few people like you, OSS projects would have many more contributors, for sure. Today, for example, it's high likely that this project lost one contributor. I wonder how many more it already lost or might lose because of behaviours like this.

**Comment by AnXh3L0** [comment]
@extremeshok ahem, after carefully reading every comment here, I came to the conclusion that this project is not worth devoting our time on and it is a clear example of why some open source projects are failing, because of people like you that do not know how to deal with new possible contributors. You are trying to stay in your comfort zone and don't like innovation and better design for your project? Be my guest, we got more important projects to contribute to instead of wasting time with projects whom communities do not welcome new contributors.

**Comment by elioqoshi** [comment]
@extremeshok Your contributions don't matter much if you don't know how to behave properly in a community. Am curious what the lead developer thinks of this. I somehow have still hope both of you don't represent the values of the project. 

Since you asked, here is the logo, together with the source files. Feel free to do whatever you want with them. Consider them Public Domain:

https://elioqoshi.me/cloud/index.php/s/KzLvRTZFHAkzRcb

![draft-01](https://cloud.githubusercontent.com/assets/5436686/25383509/b94cb106-29bc-11e7-8a04-2ac5cf3e5fa8.jpg)

You have no idea how destructive you are. You are the reason projects need Code of Conducts.

**Comment by extremeshok** [comment]
nice logos

**Comment by elioqoshi** [comment]
I work as a designer for Mozilla and Tor, trust me, I'm not doing this for my portfolio. I can imagine many other people contributing for different reasons then suggested, but you shouldn't assume my intentions are the same.

**Comment by ervee** [comment]
@AnXh3L0, extremeshok is NOT the developer or owner of this project. Don't make such a big deal of one or two random people's comments. And don't write off someone else's project because of it. Grow up!

**Comment by douglascamata** [comment]
![image](https://cloud.githubusercontent.com/assets/159076/25393251/499dae72-29db-11e7-8caf-e33ba4161ef6.png)

These comments don't look like personal opinions. And anyway, if you guys didn't have anything to contribute to a healthy discussion about the subject of the PR and are not capable of taking a decision based on it: why the hell you started to speak aggressively? 

Having an opinion on everything doesn't mean you should expose it. Sometimes, like today and here, you both could have just kept your toxic opinions to yourselves.

Be responsible and think twice before you type, stop trying to ignore consequences just `because it's my personal opinion, so I say whatever I want, herp derp`.

**Comment by ervee** [comment]
If this was a PR I'd agree. But it was an issue and it looks spammy. And I'm not going to click some link to check.

**Comment by arturbonnett** [comment]
I liked this new proposed logo / identity colors. :+1:
I think in addition to representing better, is more remarkable!
... Just my opinion, as a fan of the project.

**Comment by madpsy** [comment]
My opinion on this, for what it's worth. The OP is a tout for business no matter what 'open source' nonsense is spouted. Open Source != free (cost), especially when it comes to branding.

Looking at their site you can see that they clearly expect to be paid in money, not kind, either directly or via Patreon. Sure, the example amounts are low but it is still money, therefore this is unsolicited advertising.

A genuine contribution would be a PR with the logo they think would look nice. That's the difference between linking to a website which clearly has a 'What is your budget?' field and contributing with actual work, for free.

Receiving donations is perfectly legitimate, we all need to eat (and make a living). However, advertising your services like this is not the way to go about it and is the definition of spam.

**Comment by simonv3** [comment]
Regardless of the source of the redesign or the attitude around soliciting a redesign:

Requesting that a design change comes through a PR drives away very talented designers who aren't familiar with git. Knowing git shouldn't be a requirement for contributing to your project, especially if your skill set doesn't involve coding. 

Design is an iterative process, and you can't just "do a PR" and fix it as if it's a bug. Designers take feedback into consideration, work on it, and come back with something new. Their tools aren't all transferable to code and commits.

There are already a lot of hurdles to get designers contributing to open source, something which is a detriment to the _entire open source community_. 

But in the end this seems to be a project that harbors trolls and lets them scare off newcomers and willing contributors. @elioqoshi I honestly wouldn't waste my time.

**Comment by gusarg81** [comment]
I definitely see this as Spam. 

I am coder and designer as well, and I've worked along with some Open Source projects (like aMSN, and others in the past) and my own Open Source projects (which also, I've have my own business). So, for me to contribute to this kind of projects one should do it directly, not to post your business site... Thats clearly Spam, no matter if your business site is about Open Source or Comercial.

**Comment by brantje** [comment]
Own serval open source projects too and @elioqoshi did logo's for them, all all of them are awesome.
So imo he is not a spammer, just a great guy who like's to design logo's for free.
Other people would pay 100's if not 1000's or a logo design, so why you can't be thankful?
